modules/api/src/main/scala/vinyldns/api/domain/membership/MembershipService.scala

```diff
@@ -263,6 +265,16 @@ class MembershipService(
       .orFail(GroupNotFoundError(s"Group with ID $groupId was not found"))
       .toResult[Group]
 
+  // Validate group details. Group name and email cannot be empty
+  def groupValidation(group: Group): Result[Unit] = {
+    Option(group) match {
+      case Some(value) if value.name.isEmpty || value.email.isEmpty =>
```

Comment:
It may be a good idea to `.trim` the values in case they're whitespace.

```suggestion
      case Some(value) if value.name.trim.isEmpty || value.email.trim.isEmpty =>
```

It's also a good idea to handle the possibility of a null value:
```suggestion
      case Some(value) if Option(value.name).forall(_.trim.isEmpty) || Option(value.email).forall(_.trim.isEmpty)=>
```

Reply:
Made the suggested changes.